expect/_build_message.ts

```diff
@@ -22,17 +24,12 @@ export function buildEqualErrorMessage<T>(
 
   let message = `Values are not equal${msgSuffix}`;
 
-  try {
-    const stringDiff = (typeof actual === "string") &&
-      (typeof expected === "string");
-    const diffResult = stringDiff
-      ? diffstr(actual as string, expected as string)
-      : diff(actualString.split("\n"), expectedString.split("\n"));
-    const diffMsg = buildMessage(diffResult, { stringDiff }).join("\n");
-    message = `${message}\n${diffMsg}`;
-  } catch {
-    message = `${message}\n${red(CAN_NOT_DISPLAY)} + \n\n`;
```

Comment:
It's probably best that the underlying functions, such as `diffstr()`, `diff()`, and `buildMessage()`, throw naturally. This catch might actually impede the ability to troubleshoot.